samples/SimpleCheckout/app/src/main/java/com/shopify/checkout_sdk_sample/ProductViewModel.kt

```diff
@@ -22,20 +22,17 @@
  */
 package com.shopify.checkout_sdk_sample
 
-import android.net.Uri
 import android.util.Log
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
-import com.shopify.checkoutsheetkit.CheckoutEventProcessor
 import com.shopify.checkoutsheetkit.CheckoutException
-import com.shopify.checkoutsheetkit.pixelevents.PixelEvent
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.launch
 
-class ProductViewModel : ViewModel(), CheckoutEventProcessor {
+class ProductViewModel : ViewModel() {
```

Comment:
We need to pass an activity into the DefaultCheckoutEventProcessor constructor. So this is a bit of a refactor to move that into the view